primitives/state-machine/src/changes_trie/build.rs

```diff
@@ -666,10 +629,7 @@ mod test {
 			let (backend, storage, mut changes, config) = prepare_for_build(zero);
 
 			// 110: missing from backend, set to None in overlay
-			changes.prospective.top.insert(vec![110], OverlayedValue {
-				value: None,
-				extrinsics: Some(vec![1].into_iter().collect())
-			});
+			changes.set_storage(vec![110], None);
```

Comment:
set_extirnsic_index(1) before? (and restore afterward)

Reply:
I made it so that the extrinsic index is set to 1 now (like before). Not that it matters.